share/jupyter/hub/static/less/page.less

```diff
@@ -1,12 +1,36 @@
 .jpy-logo {
-  height: 40px;
-  margin: 8px;
+  height: 35px;
+  margin: 2px;
 }
 
-div#header {
-  border-bottom: 1px solid #ccc;
+#header {
+  border-bottom: 1px solid #e7e7e7;
+  height: 40px;
+
+
+}
+.navbar {
+  min-height:20px;
```

Comment:
definitely don't want to set a global min-height on navbar. If navbar should change height, use variables.